can/zynqmp-can/zynqmp-can-test.c

```diff
@@ -0,0 +1,401 @@
+/*
+ * Phoenix-RTOS
+ *
+ * ZynqMP CAN driver test application
+ *
+ * Copyright 2025 Phoenix Systems
+ * Author: Dariusz Sabala
+ *
+ * %LICENSE%
+ */
+
+#include <stdio.h>
+#include <errno.h>
+#include <string.h>
+#include <stdlib.h>
+#include <termios.h>
+#include <stddef.h>
+#include <sys/types.h>
+#include <phoenix/msg.h>
+#include <sys/threads.h>
+#include <sys/msg.h>
+#include <signal.h>
+#include <unistd.h>
+
+#include <zynqmp-can-if.h>
+
+/* Structure to store statistics for received frames */
+typedef struct {
+	uint32_t receivedFramesCount; /**< Number of received frames */
+	zynqmp_canFrame lastFrame;    /**< Last received frame */
+} frameStats;
+
+/* Application data */
+static struct {
+	uint8_t stack[_PAGE_SIZE] __attribute__((aligned(16)));
+	int run;                                   /**< flag to control the main loop */
+	handle_t mutex_stats_access;               /**< mutex for accessing frame statistics */
+	frameStats frame_stats[ZYNQMP_CAN_ID_MAX]; /**< Array to store statistics for each CAN ID */
+} common = {
+	.run = 1,
+};
```

Comment:
Not super important, but this one, small initialized causes whore struct to be placed in .data, along with the zeroed-out stack. It means it is a part of the ELF and has to be copied on process start ;) How about placing `common.run = 1;` in `main()`?